packages/react-spinbutton/src/components/SpinButton/useSpinButton.tsx

```diff
@@ -34,6 +36,16 @@ const MAX_SPIN_TIME_MS = 1000;
 // pull this out into a util function in the SpinButton package.
 const lerp = (start: number, end: number, percent: number): number => start + (end - start) * percent;
 
+// TODO: discuss this with design
+// const selectEntireRange = (input: HTMLInputElement | null): void => {
+//   if (!input) {
+//     return;
+//   }
+
+//   input.focus();
+//   input.setSelectionRange(0, input.value.length);
+// };
```

Comment:
Should this stay here? Seems a bit unrelated to the other changes in this PR.

Reply:
I added after discovering a bug in NVDA where it announces "blank" when spinning the value via the keyboard. @smhigley suggested I try selecting all the text in the input to get  NVDA to announce it but that didn't pan out. It's left here because it might still be a nice UX but I wanted to discuss it with design first. Happy to remove it to another branch if you want. LMK.

Reply:
Decided to just remove these and keep em around in another branch.